aksel.nav.no/website/components/sanity-modules/props/parts/DtList.tsx

```diff
@@ -1,48 +1,63 @@
-import { Detail, Tooltip } from "@navikt/ds-react";
+import dynamic from "next/dynamic";
+import { BodyShort, Skeleton } from "@navikt/ds-react";
+import { AkselTable, AkselTableRow } from "@/web/Table";
 import { Highlighter } from "./Highlight";
 
-export const DtList = ({ prop, parent }: { prop: any; parent: string }) => {
+const LazyDescription = dynamic(() => import("./DtListDescription"), {
+  ssr: false,
+  loading: () => <Skeleton />,
+});
+
+const LazyExample = dynamic(() => import("./DtListExample"), {
+  ssr: false,
+  loading: () => <Skeleton />,
+});
+
+export const DtList = ({ prop }: { prop: any; parent: string }) => {
   if (prop?.description && prop.description.includes("@private")) {
     return null;
   }
 
   return (
-    <Detail
-      as="div"
-      className="block overflow-x-auto border border-t-0 border-gray-300 p-2 first-of-type:border-t last-of-type:rounded-b"
+    <BodyShort
+      as="ul"
+      className="dtlist overflow-x-auto last-of-type:rounded-b"
     >
-      <dt>
-        {!prop.defaultValue ? (
-          <span className="font-mono font-semibold">{`${prop.name}${
-            prop?.required ? "" : "?"
-          } `}</span>
-        ) : (
-          <Tooltip
-            content={`default: ${prop.defaultValue}`}
-            arrow={false}
-            delay={0}
-          >
-            <span className="mr-2 cursor-pointer border-b border-dashed border-gray-600 font-mono font-semibold">{`${
-              prop.name
-            }${prop?.required ? "" : "?"}`}</span>
-          </Tooltip>
-        )}
-
-        <span className="font-mono">
-          {prop.type ? <>{Highlighter({ type: prop.type })}</> : ""}
-        </span>
-      </dt>
-      {prop.description && (
-        <dd className="whitespace-pre-wrap text-base">{prop.description}</dd>
+      {prop.type && (
+        <li className="my-4 flex px-3 text-base">
+          <div className="min-w-24 font-semibold">Type: </div>
+          <code className="text-sm">{Highlighter({ type: prop.type })}</code>
```

Comment:
Skulle gjerne sett at type-raden var bedre aligned. Hva med å legge på `mt-05` på `<code>`? 😇
```suggestion
          <code className="mt-05 text-sm">{Highlighter({ type: prop.type })}</code>
```
En annen mulighet er å gi de samme line-height.